app/partials/app.cjsx

```diff
@@ -3,15 +3,81 @@ auth = require 'panoptes-client/lib/auth'
 IOStatus = require './io-status'
 MainHeader = require './main-header'
 MainFooter = require './main-footer'
+GeordiClient = require 'zooniverse-geordi-client'
 {generateSessionID} = require '../lib/session'
 
+class GeordiLogger
+  constructor: (@state, @geordi) ->
+
+  keys: {}
+
+  geordi: null
+
+  instance: =>
+    @geordi = @geordi || @makeGeordi @keys?.projectToken
+
+  makeGeordi: (projectSlug) ->
+    new GeordiClient
+      server: @state?.env
+      projectToken: projectSlug || ''
```

Comment:
@eatyourgreens I agree. Geordi client was built for Ouroboros where there is only one project and it was always my intent we would update geordi client as part of this integration to meet the needs of PFE usage - this is a good example of where we need to do that, it shouldn't be bound to a project. Unfortunately I haven't been included in the development effort so this was missed. I will sync up with @wgranger and @amyrebecca and make sure we update geordi client, and this integration, accordingly.